pkg/networkservice/chains/nsmgr/suite_test.go

```diff
@@ -394,42 +397,56 @@ func (s *nsmgrSuite) Test_PassThroughRemoteUsecase() {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
 	defer cancel()
 
+	labels := []string{"a", "b", "c", "d", "e", "f", "g"}
+
+	nsReg := linearNS(labels)
+	nsReg, err := s.domain.Nodes[0].NSRegistryClient.Register(ctx, nsReg)
+	require.NoError(t, err)
+
 	var nsesReg [nodesCount]*registry.NetworkServiceEndpoint
 	for i := 0; i < nodesCount; i++ {
+		nsesReg[i] = &registry.NetworkServiceEndpoint{
+			Name:                fmt.Sprintf("endpoint-%v", labels[i]),
+			NetworkServiceNames: []string{nsReg.GetName()},
+			NetworkServiceLabels: map[string]*registry.NetworkServiceLabels{
+				nsReg.GetName(): {
+					Labels: map[string]string{
+						step: labels[i],
+					},
+				},
+			},
+		}
+
 		var additionalFunctionality []networkservice.NetworkServiceServer
-		if i != 0 {
-			// Passtrough to the node i-1
+		if i != nodesCount-1 {
 			additionalFunctionality = []networkservice.NetworkServiceServer{
```

Comment:
I don't know why original tests had an array of one element here, but we don't need this array here.
Please, replace it with one `networkservice.NetworkServiceServer element`.

Reply:
because in case of single server it would be nil if condition not met